openpype/plugins/publish/extract_review_slate.py

```diff
@@ -14,10 +14,13 @@
     get_ffmpeg_codec_args,
     get_ffmpeg_format_args,
 )
-from openpype.pipeline import publish
+from openpype.pipeline.publish import (
+    Extractor,
```

Comment:
This plugin will be discovered and processed.

Reply:
Fixed with https://github.com/ynput/OpenPype/pull/5537/commits/564cb31cbe700aef687f6b11969eb530973e60d9

Reply:
By the way @iLLiCiTiT - this particular bug actually **hangs** the publisher. The publishing just stops. It doesn't go to an error page, it doesn't go to a report page - only in the Maya script editor I see "raise ValueError". The user can stop the publisher (it doesn't hang the actual UI; it just stops doing anything remaining visually in the state as if it's still doing something).
```python
# Traceback (most recent call last):
#   File "C:\Users\Maqina-RTX-05\AppData\Local\pypeclub\openpype\3.16\openpype-v3.16.5-colorbleed.2\openpype\tools\publisher\control_qt.py", line 56, in _execute
#     item.process()
#   File "C:\Users\Maqina-RTX-05\AppData\Local\pypeclub\openpype\3.16\openpype-v3.16.5-colorbleed.2\openpype\tools\publisher\control.py", line 64, in process
#     self.callback(*self.args, **self.kwargs)
#   File "C:\Users\Maqina-RTX-05\AppData\Local\pypeclub\openpype\3.16\openpype-v3.16.5-colorbleed.2\openpype\tools\publisher\control.py", line 2519, in _process_and_continue
#     self._publish_next_process()
#   File "C:\Users\Maqina-RTX-05\AppData\Local\pypeclub\openpype\3.16\openpype-v3.16.5-colorbleed.2\openpype\tools\publisher\control.py", line 2340, in _publish_next_process
#     item = next(self._main_thread_iter)
#   File "C:\Users\Maqina-RTX-05\AppData\Local\pypeclub\openpype\3.16\openpype-v3.16.5-colorbleed.2\openpype\tools\publisher\control.py", line 2411, in _publish_iterator
#     self._publish_report.add_plugin_iter(
#   File "C:\Users\Maqina-RTX-05\AppData\Local\pypeclub\openpype\3.16\openpype-v3.16.5-colorbleed.2\openpype\tools\publisher\control.py", line 213, in add_plugin_iter
#     self._current_plugin_data = self._add_plugin_data_item(plugin)
#   File "C:\Users\Maqina-RTX-05\AppData\Local\pypeclub\openpype\3.16\openpype-v3.16.5-colorbleed.2\openpype\tools\publisher\control.py", line 229, in _add_plugin_data_item
#     raise ValueError(
# ValueError: Plugin '<class 'C:\Users\Maqina-RTX-05\AppData\Local\pypeclub\openpype\3.16\openpype-v3.16.5-colorbleed.2\openpype\plugins\publish\extract_review_slate.py.Extractor'>' is already stored
```

Seems like at least that's a case the Publisher should handle prettier. I'd expect at least the "Report your IT" message.